Core/include/Acts/Seeding/SeedFinderConfig.hpp

```diff
@@ -262,7 +262,7 @@ struct SeedFinderOptions {
     // helix radius in homogeneous magnetic field. Units are Kilotesla, MeV and
     // millimeter
     // TODO: change using ACTS units
-    options.pTPerHelixRadius = 1./0.003356 * options.bFieldInZ;
+    options.pTPerHelixRadius = 1. / 0.0033356 * options.bFieldInZ;
```

Comment:
might be worth using `Units.hpp`

Reply:
Why would we replace a constant (which I assume is supposed to be the SOL) by one over the reciprocal of that constant? If there a good reason to do this?

Reply:
so I think with `Units.hpp` it would be
```suggestion
    options.pTPerHelixRadius = UnitConstants::T * 1e-6 * options.bFieldInZ;
```
the conversion for kT and MeV looks magical but I guess this could only be improved by changing this all across the seeding code

Reply:
It is the SOL. yeah I think `UnitConstants::T * 1e6` is also OK